src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        Random randomNumber = new Random();
+        return new Ball(getRandomColor(), randomNumber.nextInt(100));
+    }
```

Comment:
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked

Reply:
My bad